src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (stylesString) => {
+  const styles = {};
+
+  const declarations = stylesString.split(';');
+
+  declarations.forEach((declaration) => {
+    const trimmedDeclaration = declaration.trim();
+
+    if (trimmedDeclaration) {
+      const [property, value] = trimmedDeclaration.split(':');
```

Comment:
There is a potential issue if the `declaration` contains more than one colon. The split will only consider the first colon, which may lead to incorrect parsing of the property or value. It would be safer to use `split(':', 2)` to ensure that only the first colon is used to split the property from the value.